ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/AddMetadata.java

```diff
@@ -5,7 +5,6 @@
 package com.mozilla.telemetry.decoder;
 
 import com.google.common.collect.ImmutableList;
-import com.mozilla.telemetry.decoder.ParseUri;
```

Comment:
all removed imports in this PR are due to being in the same package as the current file.